public/projects/dothraki_translator.md

```diff
@@ -0,0 +1,16 @@
+# Dothraki Translator
+
+![dothraki](dothraki.jpg)
+
+For the TV show _Game of Thrones_ they [hired a linguist](http://www.vulture.com/2013/04/game-of-thrones-dothraki-language-inventor.html) to create a complete fictional language for the nomadic equestrian Dothraki warrior people.
+
+Creating a full Engligh-to-Dothraki translator is a _little_ too complicated for this class. Fortunately, someone else did it, and made an API for it! 
+
+Check out <http://funtranslations.com/api/dothraki> 
+
+In this project you will make a command-line or web app that accepts user input in English, passes it to the API, parses the response, and displays the translation.
```

Comment:
This sounds fun. Is the purpose to practice AJAX and remote API messaging only? 
It could be fun to add a cache layer where if the user asks for parts of speech that have already been translated, they are returned from local storage, and the rest of the message is sent for translation in whole or in part.